UNIFY-328 Update all references to config file (#4093)
cypress-io/cypress-documentation: content/guides/guides/web-security.md

Still here? That's cool, let's disable web security!

> #### Set `chromeWebSecurity` to `false` in your [configuration file](/guides/references/configuration)

Reviewer: ```suggestion
#### Set `chromeWebSecurity` to `false` in the [Cypress configuration](/guides/references/configuration)
```
